### DISCUSSION TITLE
Colored text does not work in speech bubble

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#13936] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

‖color:gui.Blue‖‖end‖ is not converted to colored text 

### Reproduction steps

1. send this message
All leaks repaired in ‖color:gui.Blue‖command room‖end‖!

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![speechbubble](https://github.com/FakeFishGames/Barotrauma/assets/64390906/03e72153-f7cc-4ac3-b934-896ce31586ae)

**Comment by Jasontti** [comment]
Tested a fix for this, no issues found.
Color is parsed properly in chat messages.
Closing discussion as resolved.
